lib/widgets/workspace.dart

```diff
@@ -65,6 +79,25 @@ class _FilesWorkspaceState extends State<FilesWorkspace> {
     });
   }
 
+  void _setSortType(SortType? type) {
+    setState(() {
+      if (type != null) {
+        controller.sortType = type;
+        controller.columnIndex = type.index;
+        controller.changeCurrentDir(controller.currentDir);
```

Comment:
wait is there really no better way to do this? might be better to actually add a refresh mechanism because this is hacky as hell (could be my fault tho, don't remember much of this repo code)

Reply:
Exact same behavior used with `FilesTable` in `onHeaderCellTap`. Totally agree with u but for rid of this we must remake `CancelableFsFetch` because files are sorted only inside `startFetch` function

Reply:
hm that class is quite complex, i'll try figuring out something later on, for now leave it as is